app/views/users/_social.html.haml

```diff
@@ -13,17 +9,16 @@
         %p Follow
```

Comment:
This isn't a problem that is new with this PR, I'm just leaving this here for reference. If we're going to show the "Follow" button to users who aren't signed in, we should show them a prompt to sign in if they click on it. As of now, it just results in a silent 401 (unauthorized) error.